prov/rxm/src/rxm_ep.c

```diff
@@ -1058,7 +1061,8 @@ rxm_ep_inject_common(struct rxm_ep *rxm_ep, const void *buf, size_t len,
 		else if (OFI_UNLIKELY(ret != -FI_EAGAIN))
 			goto cmap_err;
 		ret = rxm_ep_postpone_send(rxm_ep, rxm_conn, NULL, 1,
-					   &iov, NULL, len, data, flags,
+					   &iov, NULL, len, data,
+					   rxm_ep_tx_flags_inject(rxm_ep) | flags,
```

Comment:
I don't understand what flags are being used here.  Why are flags being ORed with rxm_ep tx flags,   We should either use the default flags or the user flags.

Reply:
The `flags` can be set to either `0` or `FI_REMOTE_CQ_DATA`.

Reply:
> Why are flags being ORed with rxm_ep tx flags

`rxm_ep_tx_flags_inject` would get the default flags for the endpoint.